### PR TITLE
Channel config

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/channels/types/africastalking/type.py [hunk 1]
```diff
@@ -28,6 +28,59 @@ class AfricasTalkingType(ChannelType):
     claim_blurb = _("""If you are based in Kenya, Uganda or Malawi you can purchase a short
     code from <a href="http://africastalking.com">Africa's Talking</a> and connect it
     in a few simple steps.""")
+
+    configuration_blurb = _(
```
Comment (rowanseymour) [inline temba/channels/types/africastalking/type.py]: Doesn't seem a good idea having such large blobs of HTML in Pythonland. For localization this whole thing will now appear in transifex and if translators don't replicate the HTML exactly, they'll end up changing the page significantly. 

All the claim pages have a template subfolder, so why not put a real template in there, i.e. 

```
templates/channels/types/<typename>/claim.haml
templates/channels/types/<typename>/config.haml
```

Then you can be more selective about which strings are localized using the regular `-trans` tags.

@nicpottier ?

Comment (nicpottier) [reply temba/channels/types/africastalking/type.py]: Can we put the templates in the app itself? Sure would be nice to have these be totally self contained in their types folder.

Comment (rowanseymour) [reply temba/channels/types/africastalking/type.py]: Possibly - not sure how easy it is to dynamically add new folders to the template directory list and we'd want to check there aren't performance implications of having so many directories on that path. Would be neat if possible

Comment (nicpottier) [reply temba/channels/types/africastalking/type.py]: Can't imagine it is that expensive considering they allow per app templates, especially if we added them to the end of the search path so only misses higher up would get there. Believe once a template is found it is cached as well, further reducing any cost there.

Comment (nicpottier) [reply temba/channels/types/africastalking/type.py]: Seems we could just create an extension for `temba.utils.haml.HamlFilesystemLoader` that is tailored to look in channel dirs when appropriate.

@norkans7 want to give that a try?

Comment (norkans7) [reply temba/channels/types/africastalking/type.py]: Sure, I think I will do that in two steps, first using the existing template folders then move them to the types folders

Comment (nicpottier) [reply temba/channels/types/africastalking/type.py]: Agree on that approach

Comment (rowanseymour) [reply temba/channels/types/africastalking/type.py]: Well @norkans7 can experiment but I don't think we have caching enabled at the minute - I think you need to use `django.template.loaders.cached.Loader` for that and if I remember correctly there are potential implications for custom tags.

Comment (nicpottier) [reply temba/channels/types/africastalking/type.py]: Well either way, no way that is critical path for performance.

**Comment by norkans7** [comment]
Replaced by #1721

**Pull request closed without merging**